markdown/stencil-docs/reference-docs/other-objects-and-properties-overview.md

```diff
@@ -2530,7 +2530,7 @@ _These objects are called on the default `<theme-name>/templates/components/blog
 
 <b>Description:</b> The form object used to edit a customer object. <br>
 
-<b>Handlebars Expression:</b> <code>{{forms.account}}</code>
+<b>Handlebars Expression:</b> <code>{{forms.edit_account}}</code>

```

Comment:
Add the template file this can be found on. 
Also don't forget to add this PR to the ticket comments.